packages/optimizer/lib/transformers/SeparateKeyframes.js

```diff
@@ -17,6 +17,14 @@
 
 const css = require('css');
 const stringifyOptions = {indent: 0, compress: true};
+
+const allowedKeyframeProps = {
```

Comment:
It's ok to use a [Set](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=2ahUKEwjSn_Pmn7PhAhVD0aYKHQdECP0QFjAAegQIAhAB&url=https%3A%2F%2Fdeveloper.mozilla.org%2Fde%2Fdocs%2FWeb%2FJavaScript%2FReference%2FGlobal_Objects%2FSet&usg=AOvVaw2eLvvECVNKoL7bdb7l6uhS) here.

Reply:
Ok cool that's better, updated